src/views/CVERecord/SearchResults.vue

```diff
@@ -0,0 +1,300 @@
+                              </p>
+                              <p v-else>{{ description.firstChunk.concat(` ${description.secondChunk}`) }}</p>
+                              <button v-if="description.secondChunk.length > 0"
+                                @click="cveListSearchStore.handleShowMoreChange(undefined, index)"
+                                class="button is-ghost cve-is-borderless p-0"
+                              >
+                                <span v-if="!description.showMore">Show more</span>
+                                <span v-else>Show less</span>
+                              </button>
+                            </div>
+                          </div>
+                        </div>
+                      </div>
+                    </div>
+                  </div>
+                  <div id="cve-search-results-container" v-if="cveListSearchStore.totalSearchResultCount > 0">
+                    <h2 class="title">Search Results</h2>
+                    <div class="mt-2 mb-2">
+                      <p>
+                        <span>Showing </span>
+                        <span>
+                          <span v-if="parseInt(cveListSearchStore.pagination.toCount) === 1" class="has-text-weight-bold">1</span>
+                          <span v-else class="has-text-weight-bold">
+                            <span class="has-text-weight-bold">
+                              {{ cveListSearchStore.pagination.fromCount.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',') }}
+                            </span>
+                            <span v-if="typeof cveListSearchStore.pagination.toCount !== 'undefined'" class="has-text-weight-bold">
+                              -
+                              {{ cveListSearchStore.pagination.toCount.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',') }}
+                            </span>
+                          </span>
+                        </span>
+                        <span> of </span>
+                        <span class="has-text-weight-bold">
+                          {{ cveListSearchStore.totalSearchResultCount.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',') }}
+                        </span>
+                        <span> results for </span>
+                        <span class="has-text-weight-bold">{{ cveListSearchStore.query }}</span>
+                      </p>
+                    </div>
+                    <div class="field is-grouped is-grouped-right pb-3" style="border-bottom: 2px solid #e5e5e5;">
+                      <label class="label">Show:</label>
+                      <div class="control">
+                        <div class="select">
+                          <select v-model="cveListSearchStore.pagination.numberPerPage" aria-label="Select how many search results to show">
+                            <option v-for="num in cveListSearchStore.pagination.numberPerPageOptions" :value="num">{{ num }}</option>
```

Comment:
This should have a `:key` value. Maybe just the index from the for loop. It helps Vue render more efficiently